include/osquery/core.h

```diff
@@ -92,10 +92,47 @@ osquery::QueryData query(const std::string& q, int& error_return, sqlite3* db);
  * virtual tables attached. This method is the main abstraction for creating
  * SQLite3 databases within osquery.
  *
+ * Note: osquery::initOsquery must be called before calling createDB in order
+ * for virtual tables to be registered.
+ *
  * @return a SQLite3 database with all virtual tables attached
  */
 sqlite3* createDB();
 
+
+/**
```

Comment:
Is `core.h` the appropriate place to put these? I was confused by the fact that there's only one function in `sqlite_util.h`.